### PR TITLE
Added support for normal and achromatic beamforming at LWA-SV

### PR DESCRIPTION
**Pull request by cdilullo**

### CODE REVIEW

**Comment by jaycedowell** [comment]
Oh, it's failing in Travis.

File: DRX/Fringing/fringeBeam.py [hunk 1]
```diff
@@ -279,6 +282,8 @@ def main(args):
                         help='dipole number')
     parser.add_argument('filename', type=str, nargs='+',
                         help='filename to process')
+    parser.add_argument('-S','--lwasv', action='store_true',
+                        help='Station is LWA-SV')
```
Comment (jaycedowell) [inline DRX/Fringing/fringeBeam.py:286]: I usually use either `-s` (lower cased) if I can or a `-v` to switch over to LWA-SV in a script.

File: DRX/Fringing/fringeSDF.py [hunk 2]
```diff
@@ -219,14 +232,16 @@ def main(args):
                         help='reference dipole for the fringing')
     parser.add_argument('-b', '--dp-beam', type=aph.positive_int, default=2,
                         help='DP beam to run the observation on')
-    parser.add_argument('-l', '--obs-length', type=aph.positive_float, default=3600.0,
+    parser.add_argument('-l', '--duration', type=aph.positive_float, default=3600.0,
```
Comment (jaycedowell) [inline DRX/Fringing/fringeSDF.py:235]: What's wrong with `--obs-length`?

File: DRX/Fringing/fringeSDF.py [hunk 3]
```diff
@@ -128,7 +134,14 @@ def main(args):
...
         
-        gains = [[twoByteSwap(gain_to_dpg(g)) for g in baseBeamGain] for i in xrange(260)] # initialize gain list
+        gains = [[twoByteSwap(gain_to_dpg(g)) for g in baseBeamGain] for i in xrange(int(len(antennas)/2))] # initialize gain list 
+        #If achromatic beamforming is requested, set the gains for it.
+        if args.custom:
+            #These weird gain values will signal to ADP to use the predetermined
+            #achromatic beam gains located in /home/adp/.
+            gains[0] = [8191,16383,32767,65535]
+            gains[1] = [1,1,1,1]
+
```
Comment (jaycedowell) [inline DRX/Fringing/fringeSDF.py:144]: I'm not sure about adding an option for custom beamforming in this script.  It's not very general purpose at this stage and it's completely undocumented.

File: DRX/Fringing/simulateFringesBright.py [hunk 4]
```diff
@@ -144,5 +147,14 @@ def main(args):
...
+    description='Simulate fringes for a dipole-dipole data set using the lsl.sim.vis.build_sim_data() function and the bright sources listed in lsl.sim.vis.srcs.',
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+    parser.add_argument('filenames', type=str, nargs='+',
+            help='Data sets (.npz)')
+    parser.add_argument('-S','--lwasv', action='store_true',
+            help='Station is LWA-SV')
 
+    args = parser.parse_args()
+    main(args)
```
Comment (jaycedowell) [inline DRX/Fringing/simulateFringesBright.py:160]: Good call on adding in `argparse`.

File: DRX/Fringing/fringeBeam.py (hunk 1 above)
Comment (jaycedowell) [inline DRX/Fringing/fringeBeam.py:286]: I usually use either `-s` (lower cased) if I can or a `-v` to switch over to LWA-SV in a script.

**Comment by jaycedowell** [comment]
[Travis is still failing.](https://travis-ci.org/github/lwa-project/commissioning/jobs/755268760)

**Comment by jaycedowell** [comment]
Hey, it's passing now!

**Comment by cdilullo** [comment]
I'm working on fixing the issues with reading drx files from SV

**Comment by cdilullo** [comment]
Ok, I think everything is working now based on my most recent data.

**Comment by jaycedowell** [comment]
You know, the reading may be even easier with `lsl.reader.ldp.LWADataFile`.

**Pull request merged**